core/frontend/src/App.vue

```diff
@@ -62,15 +62,21 @@
     <v-main>
       <router-view />
     </v-main>
+    <services-fetcher />
   </v-app>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
+import ServicesFetcher from './components/servicesFetcher.vue'
 
 export default Vue.extend({
   name: 'App',
 
+  components: {
+    'services-fetcher': ServicesFetcher,
+  },
+
```

Comment:
I have a small problem with the 'ServicesFetcher' name. It seems like it's responsible for fetching service data, not info about the service. Something like 'ServiceInfoFetcher' wouldn't be better?

Reply:
Yeah... let's see if I come up with a better name overnight...